spec/support/capybara.rb

```diff
@@ -34,3 +34,18 @@
 
 Capybara.javascript_driver = ENV.fetch("CAPYBARA_DRIVER", "headless_chrome").to_sym
 Capybara.default_driver = ENV.fetch("CAPYBARA_DRIVER", "headless_chrome").to_sym
+
+# This will fail feature specs on javascript errors/warnings
+RSpec.configure do |config|
+  config.after(:each, type: :feature) do
+    errors = page.driver.browser.manage.logs.get(:browser)
+
+    if errors.present?
+      aggregate_failures "javascript errors/warnings" do
+        errors.each do |error|
+          raise error.message
+        end
+      end
+    end
+  end
+end
```

Comment:
I love this and want it in every project 🥇

Reply:
Or maybe this:
```
      aggregate_failures "javascript errors" do
        errors.each do |error|
          expect(error.level).not_to eq("SEVERE"), error.message
        end
      end
```

Reply:
Often you can't avoid warnings

Reply:
We discussed that, but I'm not sure that I agree, I don't remember a lot of js warnings that should be ignored.
Maybe you have a recent example? :-)

Reply:
There are these levels: Verbose, Info, Warning, or Error.

I assume you wouldn't want it to block on verbose and info messages.

Then do this:
```
          expect(error.level).not_to eq("SEVERE"), error.message
          expect(error.level).not_to eq("WARNING"), error.message
```

Then it is easy to remove the warning line, if it starts acting up (it will, if you add Stripe Checkout).

Reply:
FYI we did try with `console.log` and `console.info` to check if we could get rid of catch accidental console.log calls.

But the returned errors unfortunately did not seem to include `log` and `info` levels